グループ作成機能の実装

ao-midori opened the pull request:
# WHAT
- グループ作成機能追加
- グループ作成周りのビューファイルの書き換え（form_forへ書き換え）
- （gem 'pry-rails'インストール）

# WHY
- サービス必須機能のため

ao-midori committed:
create pull request

ao-midori committed:
Added default value to some columns with not-null regulation

ao-midori committed:
Installed  gem 'pry-rails'

ao-midori committed:
Enabled to create a group, edited view files describing with form_for

b-i-hide commented on app/controllers/groups_controller.rb:
```diff
@@ -2,9 +2,16 @@ class GroupsController < ApplicationController
   end
 
-  def edit
+  def create
```
#createは成功した時と失敗した時で処理を分けましょう！

ao-midori replied:
グループ名にバリデーションをかけて、ブランクで登録不可にしました。
create失敗時には、エラーメッセージを表示させるようにしました。
create成功時には、チャットページへ変遷します。

ao-midori replied:
<img width="1280" alt="2017-02-16 22 45 19" src="https://cloud.githubusercontent.com/assets/25502276/23023600/bcac89fa-f499-11e6-9295-c3db1364bbba.png">

b-i-hide commented on app/views/groups/new.html.haml:
```diff
@@ -1,12 +1,11 @@
+  = form_for(@group) do |f|
 
-    = render partial: 'group'
+    = render partial: 'group', locals: { f: f }
```
部分テンプレートを使おうとしていていいですね！
ただ、折角部分テンプレートを使っているので、下のsubmitなどもしっかりと部分テンプレートに含めましょう！

ao-midori replied:
submit部分も部分テンプレート(_group.html.haml)へ入れました。

ao-midori committed:
Cannot create group without entering group name

If success to create a new group, link to chat-page. Otherwise, showing
notice message.

ao-midori committed:
Made another ya.yml and, fixed messages layout

ao-midori committed:
Put f.submit black into partial template from group/new.html.haml
